src/ff7_opengl.cpp

```diff
@@ -31,6 +31,12 @@
 
 unsigned char midi_fix[] = {0x8B, 0x4D, 0x14};
 WORD snowboard_fix[] = {0x0F, 0x10, 0x0F};
+byte y_pos_offset_display_damage_30[] = {0, 1, 2, 3, 4, 5, 6, 6, 7, 7, 8, 8, 8, 8, 7, 7, 6, 6, 5, 4, 3, 2};
+//byte y_pos_offset_display_damage_60[] = {0, 0, 1, 1, 2, 2, 3, 3, 4, 4, 5, 5, 6, 6, 6, 6, 7, 7, 7, 7, 8, 8,
```

Comment:
I left this one commented since this is the actual interpolation, but I changed it to make it smoother. Don't really know which one is better.

Reply:
Can you please put those lines into a single one instead of two? Thank you